rasterizer/renderer.go

```diff
@@ -107,8 +107,9 @@ func (r *Renderer) RenderImage(img image.Image, m canvas.Matrix) {
 	// add transparent margin to image for smooth borders when rotating
 	margin := 4
 	size := img.Bounds().Size()
+	sp := img.Bounds().Min // starting point
 	img2 := image.NewRGBA(image.Rect(0, 0, size.X+margin*2, size.Y+margin*2))
-	draw.Draw(img2, image.Rect(margin, margin, size.X, size.Y), img, image.Point{}, draw.Over)
+	draw.Draw(img2, image.Rect(margin, margin, size.X+margin, size.Y+margin), img, sp, draw.Over)
```

Comment:
Nice find, I forgot that the 3rd and 4th parameters are coordinates, not width/height!